api/scripts/getOldPolkadotRewardEras.js

```diff
@@ -332,13 +344,29 @@ async function main() {
   )
   console.timeEnd('parsing lunie rewards')
   
+  // console.log(`lunieRewards: `, lunieRewards)
+
   // store
-  const storableRewards = _.uniqBy(lunieRewards
-    ? lunieRewards.filter(({ amount }) => amount > 0)
-    : [], reward => `${reward.address}_${reward.validator}_${reward.height}_${reward.chain_id}`) // HACK somehow we get some rewards twice which causes the insert to fail 
+  // const storableRewards = _.uniqBy(lunieRewards
+  //   ? lunieRewards.filter(({ amount }) => amount > 0)
+  //   : [], reward => `${reward.address}_${reward.validator.id}_${reward.height}_${reward.chain_id}`) // HACK somehow we get some rewards twice which causes the insert to fail 
+  
+  let storableRewards = []
+  lunieRewards.filter(({ amount }) => parseFloat(amount) > 0).forEach(reward => {
+    if (!storableRewards.find(storableReward => storableReward.address === reward.address && storableReward.validatorAddress === reward.validatorAddress && storableReward.height === reward.height && storableReward.chain_id === reward.chain_id )) {
```

Comment:
can you comment what this check does?

Reply:
Removed, as I dont see duplicates in results.